### PR TITLE
Create AccessRights domain

### LINKED ISSUE #446: Create AccessRights Domain
Create AccessRights Domain with the following resources:

- AccessRight
- all the checks

### PR DESCRIPTION
**Pull request by kagure-nyakio**
Fixes #446

### CODE REVIEW

File: lib/omedis/access_rights/resource_access_rights.ex [hunk 1]
```diff
@@ -1,4 +1,4 @@
-defmodule Omedis.Accounts.Relationships.ResourceAccessRights do
+defmodule Omedis.AccessRights.ResourceAccessRights do
```
Comment (almirsarajcic) [inline lib/omedis/access_rights/resource_access_rights.ex:1]: Inconsistent. Aren't we using the `Relationships` namespace?
https://github.com/3Stones-io/omedis/blob/dd024a1e13bedb53b2c4cd0badd382ff23e4f080/lib/omedis/invitations/invitation/relationships/invitation_access_rights.ex#L1

File: priv/repo/demo_seeds.exs [hunk 2]
```diff
@@ -47,7 +49,7 @@ end
 
 %{records: [group_1, group_2], status: :success} =
   bulk_create.(
-    Accounts.Group,
+    Groups.Group,
```
Comment (almirsarajcic) [inline priv/repo/demo_seeds.exs:52]: Well, if you didn't change it in the appropriate PR, don't change it now. When you start working on fixing seeds task, you can do it there.

File: lib/omedis/accounts/checks/group_and_projects_exist.ex [hunk 3]
```diff
@@ -1,21 +0,0 @@
-defmodule Omedis.Accounts.Checks.GroupAndProjectExist do
```
Comment (almirsarajcic) [inline lib/omedis/accounts/checks/group_and_projects_exist.ex:1]: What's with this?

**Review by almirsarajcic — CHANGES_REQUESTED** [review]

File: lib/omedis/accounts/checks/group_and_projects_exist.ex (hunk 3 above)
Comment (kagure-nyakio) [reply lib/omedis/accounts/checks/group_and_projects_exist.ex:1]: It is a check that is never used.

File: lib/omedis/access_rights/resource_access_rights.ex (hunk 1 above)
Comment (kagure-nyakio) [reply lib/omedis/access_rights/resource_access_rights.ex]: Updated

**Review by almirsarajcic — APPROVED** [review]
Great, thanks!

**Pull request merged**